engine/src/main/java/com/redhat/cloud/notifications/events/EventConsumer.java

```diff
@@ -114,24 +105,7 @@ public CompletionStage<Void> process(Message<String> message) {
             bundleName[0] = action.getBundle();
             appName[0] = action.getApplication();
             String eventTypeName = action.getEventType();
-            Log.infof("Processing received action (id=%s): (accountId=%s, orgId=%s) %s/%s/%s", action.getId(), action.getAccountId(), action.getOrgId(), bundleName[0], appName[0], eventTypeName);
-            if (action.getOrgId() == null || action.getOrgId().isBlank()) {
-                String bundle = bundleName[0] == null ? "" : bundleName[0];
-                String application = appName[0] == null ? "" : appName[0];
-                Counter missingOrgIdCounter = registry.counter(MISSING_ORG_ID, "bundle", bundle, "application", application);
-                missingOrgIdCounter.increment();
-                if (featureFlipper.isUseOrgId()) {
-                    Log.info("The org ID migration is enabled but the orgId field is missing or blank in the action");
-                }
-                if (featureFlipper.isTranslateAccountIdToOrgId() && action.getAccountId() != null && !action.getAccountId().isBlank()) {
-                    try {
-                        String orgId = orgIdTranslator.translate(action.getAccountId());
```

Comment:
We didn't translate anything since August 22 on prod. This can be safely removed.